lib/tasks/slugs.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :slugs do
+  desc "Generate any missing slug string for objects created before the Sluggable concern"
+  task generate_missing: :environment do
+    Rails.application.eager_load!
+    class_names = ActiveRecord::Base.descendants.select{|c| c.included_modules.include?(Sluggable)}.map(&:name)
```

Comment:
I really enjoyed finding out this one :), without `Rails.application.eager_load!` it doesn't work, but I'm worried it won't be there for Rails 5. Anyway forks that just get this along with Rails 5 update.. may have to find their way manually backfilling slugs. This commodity is just for the cool ones that are updated :)